scripts/create_project_info.py

```diff
@@ -81,7 +81,7 @@ def build_number():
         return str(build_number)
 
     print('Buildkite build # not found, using dev alternative')
-    alt_build_number = datetime.now().strftime('%y%m%d%H%M')
+    alt_build_number = (int(datetime.now().strftime('%y%m%d%H%M')) - build_base_number) * 2 + increment_for_64bit
```

Comment:
Now that we're in '22, it was overflowing the 2,147,483,647 max int size. Started it from a smaller base so it will still increment, but without overflowing. These versions are just for development builds.